codalab/scripts/initialize.py

```diff
@@ -168,6 +169,24 @@ def insert_data():
     for name, codename in submission_status_set:
         _, _ = CompetitionSubmissionStatus.objects.get_or_create(name=name, codename=codename)
 
+    team_status = [("Denied", "denied", "Team was denied."),
```

Comment:
I will do this, just don't let me forget:

1. [ ] Remove this code and initialize_teams
2. [ ] Create data migration in teams app for this
3. NOTE: If you need to run this code on a database where this initialization has already been ran, can fake the migration